ideas/99-confidence.md

```diff
@@ -0,0 +1,121 @@
+## Preamble
+
+    Idea: 99
+    Title: Confidence in Messaging
+    Status: Draft
+    Created: 2018-03-22
+
+
+## Summary
+As end users we currently don't have confidence that messages are being sent, delivered and read. This swarm aims to address this.
+
+## Swarm Participants
+- Lead Contributor: @rasom
+- Testing & Evaluation: <!-- @username -->
+- Contributor: <!-- @username -->
```

Comment:
IIRC:

1 Clojure
1 Go
1 Design
1 UXR
1 PM

Missing / too much?

Reply:
What is minimum to get started?

Reply:
probably 1 QA? but otherwise it's enough